docs/rules/jsx-space-before-closing.md

```diff
@@ -1,8 +1,10 @@
 # Enforce spacing before closing bracket in JSX (react/jsx-space-before-closing)
 
+❌ This rule is deprecated.
+
 🔧 This rule is automatically fixable using the `--fix` [flag](https://eslint.org/docs/latest/user-guide/command-line-interface#--fix) on the command line.
 
-**Deprecation notice**: This rule is deprecated. Please use the `"beforeSelfClosing"` option of the [jsx-tag-spacing](https://github.com/jsx-eslint/eslint-plugin-react/blob/master/docs/rules/jsx-tag-spacing.md) rule instead.
+Please use the `"beforeSelfClosing"` option of the [jsx-tag-spacing](https://github.com/jsx-eslint/eslint-plugin-react/blob/master/docs/rules/jsx-tag-spacing.md) rule instead of this rule.
```

Comment:
it seems like this comment should be part of the deprecation line, instead of separate.

Reply:
The test currently checks that the generic deprecation notice is on its own line which is why I moved the rule-specific notice to a separate line. If you still want it to be on the same line, we could figure out some workaround.

Reply:
That would be great.

Reply:
Extracted to https://github.com/jsx-eslint/eslint-plugin-react/pull/3364